### ISSUE TITLE
About Dataset_Custom

### ISSUE THREAD

**Issue by Ray9uaza** [thuml/Time-Series-Library#111]
Can I remove the validation set when using `Dataset_Custom` for prediction tasks and only use the training and test sets? If yes, how can I do it?
